### ISSUE TITLE
Improve message for sniff for constants (rule 107)

### ISSUE THREAD

**Issue by krisdante** [magento/magento-coding-standard#163]
### Description
https://github.com/magento/magento-coding-standard/issues/107 message says:

`Constants must have short description if they add information beyond what the constant name supplies.`

This is missleading message as this warning is shown for all the constants. Even obvious ones as `NUMBER_PI=3.14`.

### Expected behavior
The message shall say: 
`Constants must have short description`

### Benefits
Less missleading

**Comment by lenaorobei** [comment]
Hi @krisdante. Could you please share the whole code snipped including PHP DocBlock?
Let me share the expected behavior of this rule.

**Not OK** (Description duplicates the name):
``` php
/**
 * Nesting separator.
 */
const NESTING_SEPARATOR = '->';
```
**OK** (no description):
``` php
const NESTING_SEPARATOR = '->';
```
**OK** (meaningful description):

```php
/**
 * Some additional description beyond what the constant name supplies.
 */
const NESTING_SEPARATOR = '->';
```

**Comment by krisdante** [comment]
Yes, you are right. I got confused by the message.
